js/muzima.js

```diff
@@ -163,6 +163,9 @@ $(document).ready(function () {
             validForm = validForm && $.fn.customValidationCheck();
         }
         if (validForm) {
+	    $(this).find('input:text').each(function(){
```

Comment:
@nribeka the line $(this).find('input:text').each(function() is required to trim the leading & trailing whitespaces for input text elements & this was for MUZIMA-325.